@xen-orchestra/backups/_VmBackup.js

```diff
@@ -173,7 +173,11 @@ class VmBackup {
     const settings = this._settings
 
     const doSnapshot =
-      this._isDelta || (!settings.offlineBackup && vm.power_state === 'Running') || settings.snapshotRetention !== 0
+      this._isDelta ||
+      (!settings.offlineBackup && vm.power_state === 'Running') ||
+      settings.snapshotRetention !== 0 ||
+      // https://help.vates.fr/#ticket/zoom/6735
+      settings.unconditionalSnapshot
```

Comment:
Move this as first condition, no need to do any other tests if it's enabled :slightly_smiling_face: